pocs/tests/test_arduino_io.py

```diff
@@ -76,27 +79,101 @@ def receive_message_with_timeout(subscriber, timeout_secs=1.0):
             return None, None
 
 
+@contextlib.contextmanager
```

Comment:
Wouldn't a normal fixtgure do the same thing if you `yield` it?

Reply:
Yes, but I wouldn't be able to pass in the args.